app/assets/stylesheets/shared/bootstrap-extensions.css.scss

```diff
@@ -10,3 +10,35 @@
     }
   }
 }
+
+#chat-with-us {
+  margin-bottom: 40px;
+
+  &--heading {
```

Comment:
I have an question. I can't find this id `#chat-with-us--heading` be used in the html, are there something I misunderstanding?